### DISCUSSION TITLE
Runelite crashing

### DISCUSSION THREAD

**Discussion by liamwilcke** [runelite/runelite#18152] [Q&A]
Hello.

About every 3 hours my Runelite crashes, this has been happening for roughly 2 weeks now. When this happens it's either Runelite fully freezing, while the client is still visible, or the client instantly closes itself. I never had any issues prior, i'm on an extremely good pc and never have any similar problems with any other games. I tried removing all my external plugins, then tried running in safe mode, but even in safe mode nothing changes. Also removed and reinstalled runelite, didn't fix the issue either.

If anyone has suggestions would love to hear it.

**Comment by cdfisher** [comment]
Can you upload your `client.log` and any recent `jvm_crash` logs?

To find logs, do one of the following things:

Open the screenshot directory (if you have access to client by right-clicking "Camera" button) and navigate 1 directory up and then open logs folder
Navigate to %userprofile%\.runelite\logs on Windows or $HOME/.runelite/logs on Linux and macOS.

**Reply by liamwilcke** [reply]
Hi, here client log and some recent crashes:
[client.log](https://github.com/user-attachments/files/16570226/client.log)
[jvm_crash_pid_10164_r.log](https://github.com/user-attachments/files/16570234/jvm_crash_pid_10164_r.log)
[jvm_crash_pid_6780_r.log](https://github.com/user-attachments/files/16570237/jvm_crash_pid_6780_r.log)
[jvm_crash_pid_21688_r.log](https://github.com/user-attachments/files/16570238/jvm_crash_pid_21688_r.log)

**Reply by cdfisher** [reply]
> Hi, here client log and some recent crashes: [client.log](https://github.com/user-attachments/files/16570226/client.log) [jvm_crash_pid_10164_r.log](https://github.com/user-attachments/files/16570234/jvm_crash_pid_10164_r.log) [jvm_crash_pid_6780_r.log](https://github.com/user-attachments/files/16570237/jvm_crash_pid_6780_r.log) [jvm_crash_pid_21688_r.log](https://github.com/user-attachments/files/16570238/jvm_crash_pid_21688_r.log)

It's probably #10628. If you look near the bottom of the thread, there are some steps you can take that generally resolve this issue for most people.
